erpnext_com/www/docs/user/manual/en/setting-up/automation/auto-repeat.md

```diff
@@ -0,0 +1,71 @@
+
+For Example: Assuming that you follow deferred expense system for some items. It requires you to create same Journal Entry every month to credit Deferred Expense account and debit Expense Account. You can create first Journal Entry manually for it, and then create auto-repeat transaction for it.
+
+### 1. How to set up Auto Repeat
+
+#### 1.1 Customize the Form
+1. Go to: **Customization > Form Customization > Customize Form**.
+2. Select the Form in which you want to allow creation of repeatable documents.
+3. Check `Allow Auto Repeat` to allow the creation of repeatable documents for that Form.
+
+<img class="screenshot" alt="Allow Auto Repeat" src="/docs/assets/img/setup/automation/allow-auto-repeat.png">
+
+#### 1.2 Set up Auto Repeat
+1. Go to **Settings > Auto Repeat > New**.
+2. Select the Reference Document Type.
+3. Select the Reference Document.
+4. Set the Start Date and End Date (optional). 
+If End Date is not specified, recurring documents will be created, unless the Auto Repeat is disabled.
+5. Set the Frequency for creating repeatable documents 
+(Daily, Weekly, Monthly, Quarterly, Half-yearly, Yearly).
+6. Save.
+
+#### 1.3 Set up Auto Repeat directly from the document
+You can also set a document on Auto Repeat by clicking the **Repeat** option from the **Menu** in the Toolbar.
+
+**Note**: If a document is already on Auto Repeat, the Repeat option is not available.
+
+<img class="screenshot" alt="Allow Auto Repeat" src="/docs/assets/img/setup/automation/repeat-option.png">
+
+Once you click on Repeat, a prompt for Auto Repeat will show up. Fill in the details and click on Save.
+
+<img class="screenshot" alt="Auto Repeat Prompt" src="/docs/assets/img/setup/automation/auto-repeat-prompt.png">
+
+### 2. Features
+
+#### 2.1 Notify by Email
+If you want to notify certain contacts whenever the recurring documents are created, you can check `Notify by Email` in the Notification section of Auto Repeat. Fields for the same are explained below:
+
+- **Recipients**: Defines the Email Ids of the recipients for recurring document creation emails.
+- **Get Contacts**: This button will fetch the contacts linked to the document that is set on Auto Repeat and fill up the Recipients field with the same.
+- **Template**: You can choose an Email Template for the email. This will fill up the Subject and Message fields as well.
+- **Subject**: Subject for your Email (example: Recurring ToDo created successfully).
+- **Message**: Message to be sent in the Email.
+- **Preview Message**: This button will show a preview of the message.
+- **Print Format**: Select a print format to define document view which should be emailed to customer.
+
+#### 2.2 Repeat on Day
+If the frequency is set as Monthly, Quarterly, Half-yearly or Yearly, then it will create recurring documents in the respective months on the same day as the `Start Date` of Auto Repeat. If you want to create recurring documents on some other day then you can set one of the following:
```

Comment:
```suggestion
If the frequency is set as Monthly, Quarterly, Half-yearly or Yearly, then it will create recurring documents in the respective months on the same day as the 'Start Date' of Auto Repeat. If you want to create recurring documents on some other day then you can set one of the following:
```